src/types/inventoryTypes/commonInventoryTypes.ts

```diff
@@ -112,6 +113,7 @@ export interface IEquipmentDatabase {
     Expiry?: IMongoDate;
     SkillTree?: string;
     ArchonCrystalUpgrades?: IArchonCrystalUpgrade[];
+    ItemId: IOid;
```

Comment:
_id is converted to ItemId when converting a mongoose document to a POJO. (toJSON()).